src/runtime/styled.js

```diff
@@ -33,13 +71,9 @@ export const styled = (type, displayName, styles, getStyles) => {
     : Styled;
 
   decorated.withComponent = nextType =>
-    styled(nextType, displayName, styles, getStyles);
+    styled(nextType, displayName, styles, camelName, kebabName);
 
   return decorated;
-};
+}
 
 export default styled;
-
-export const css = () => {
```

Comment:
bleh i didn't realize the docs suggested using this in all cases. should i add it back?

Reply:
yes please

Reply:
ping